index.bs

```diff
@@ -2227,6 +2227,46 @@ During the above process, the user agent SHOULD show some UI to the user to guid
 authorizing an authenticator. When <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}, prominent modal UI should <i>not</i> be shown <i>unless</i> credential creation was previously consented to via means determined by the user agent.
 </div>
 
+#### Registration API Exceptions #### {#sctn-registration-api-exceptions}
+
+[=[WRPS]=] can encounter the following {{DOMException|DOMExceptions}} from a call to {{CredentialsContainer/create()|navigator.credentials.create()}}. Some errors can have multiple reasons for why they happened, requiring the [=[WRPS]=] to infer the actual reason based on their use of WebAuthn:
```

Comment:
Let's mark this section non-normative.

Reply:
Addressed in https://github.com/w3c/webauthn/pull/2047/commits/6760c571dc51670d256ddbc37fd4ddf8dda8b8c0 ✌️